obs-browser-plugin.cpp

```diff
@@ -126,6 +126,27 @@ static void browser_source_get_defaults(obs_data_t *settings)
 	obs_data_set_default_bool(settings, "reroute_audio", false);
 }
 
+static void browser_source_get_defaults_v2(obs_data_t *settings)
+{
+	struct obs_video_info ovi;
+	obs_get_video_info(&ovi);
+
+	obs_data_set_default_string(settings, "url",
+				    "https://obsproject.com/browser-source");
+	obs_data_set_default_int(settings, "width", ovi.base_width);
+	obs_data_set_default_int(settings, "height", ovi.base_height);
+	obs_data_set_default_int(settings, "fps", 30);
```

Comment:
Would it also make sense to default to the canvas FPS here?

Reply:
This fps setting is only used if 'use custom fps' is checked.

Reply:
I'm aware of that, but in those situations wouldn't it also make sense to default to the current FPS?